shared/responses/delete.yml

```diff
@@ -0,0 +1,13 @@
+description: Returns true if the delete was successful
+
+content:
+  application/json:
+    schema:
+      type: object
+      properties:
+        id:
+          schema:
+            $ref: ../resources/addresses/attributes/adr_id.yml
+        message:
```

Comment:
Cancel P/D/L/C uses the same response object, so let's put the whole response in `shared/responses` as `deleted.yml`.

The deleted response object needs a little love, too:

```
properties:
  id:
    $ref: ../resources/addresses/attributes/adr_id.yml 
  deleted:
    type: boolean
    required: true
    example: true
```

Reply:
If we look at https://docs.lob.com/#addresses_delete it shows the response as

```
{
  "id": "adr_43769b47aed248c2",
  "deleted": true
}
```
The property `deleted` is a `boolean` with value `true`, not a `string` with `value` "deleted: true". 

I realize that this stuff is squirrely. Being able to run prism makes all the difference.